myhpi/core/templatetags/core_extras.py

```diff
@@ -53,6 +53,16 @@ def tag_external_links(content):
     return template.render(Context())
 
 
+@register.filter(name="touchify_abbrevations")
+def touchify_abbrevations(content):
+    """Takes the content of a website and inserts external link icons after every external link."""
+    external_links = re.finditer("<abbr[^>]*>[^<]*", content)
```

Comment:
these are not external links

Reply:
Argh, silly me didn't rename anything. Sorry.